### PR TITLE
Debris disc fitter, paper version

### PR DESCRIPTION
**Pull request by rbooth200**
This contains the code needed to update the debris disc fitter routines to the versions in the published paper. There are a few other changes in this branch unfortunately, but they are cosmetic or additional minor features, e.g. adding direct interpolation formulae to the Hankel transform.

### CODE REVIEW

File: frank/hankel.py [hunk 1]
```diff
@@ -206,6 +206,66 @@ def coefficients(self, q=None, direction='forward'):
...
+        -------
+        Y : array, size = (len(q), N)
+            The interpolation matrix
+        """
+        if space == 'Real':
+            x = np.atleast_1d(2*np.pi * q * self._Qmax)
+        elif space == 'Fourier':
+            x = np.atleast_1d(2*np.pi * q * self._Rmax)
+        else:
+            raise ValueError("Space must be one of 'Real' or 'Fourier, not "
```
Comment (jeffjennings) [inline frank/hankel.py:233]:
```suggestion
            raise ValueError("Space must be one of 'Real' or 'Fourier', not "
```

File: frank/statistical_models.py [hunk 2]
```diff
@@ -18,6 +18,7 @@
 #
 
 
+import multiprocessing
```
Comment (jeffjennings) [inline frank/statistical_models.py:21]:
```suggestion
# import multiprocessing
```

**Review by jeffjennings — CHANGES_REQUESTED** [review]
I've reviewed this, but you might want to look it over once more too.

**Comment by jeffjennings** [comment]
Along with merging this, we could also delete the `debris` branch (it no longer has any commits ahead of `master`).

**Comment by rbooth200** [comment]
I accepted your changes and have checked over the code. Let me know if there is anything else. Otherwise, feel free to merge this and delete the debris* branches

**Review by jeffjennings — APPROVED** [review]

**Pull request merged**